src/unit-taxonomy-tags-drawer/TagBubble.test.jsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+import { IntlProvider } from '@edx/frontend-platform/i18n';
+import { initializeMockApp } from '@edx/frontend-platform';
+import { AppProvider } from '@edx/frontend-platform/react';
+import { render } from '@testing-library/react';
+import PropTypes from 'prop-types';
+
+import initializeStore from '../store';
+
+import TagBubble from './TagBubble';
+
+let store;
+
+const data = {
+  value: 'Tag 1',
+};
+
+const TagBubbleComponent = ({ value, subTagsCount, implicit }) => (
+  <AppProvider store={store}>
+    <IntlProvider locale="en" messages={{}}>
+      <TagBubble value={value} subTagsCount={subTagsCount} implicit={implicit} />
+    </IntlProvider>
+  </AppProvider>
+);
+
+TagBubbleComponent.defaultProps = {
+  subTagsCount: 0,
+  implicit: true,
+};
+
+TagBubbleComponent.propTypes = {
+  value: PropTypes.string.isRequired,
+  subTagsCount: PropTypes.number,
+  implicit: PropTypes.bool,
+};
+
+describe('<TagBubble />', async () => {
```

Comment:
```suggestion
describe('<TagBubble />', () => {
```

This is giving a warning:

```
        Returning a Promise from "describe" is not supported. Tests must be defined synchronously.
        Returning a value from "describe" will fail the test in a future version of Jest.
```